packages/codebytes/package.json

```diff
@@ -39,7 +43,9 @@
   "devDependencies": {
     "@emotion/jest": "^11.3.0",
     "@testing-library/dom": "^7.31.2",
-    "@testing-library/react": "^11.0.4"
+    "@testing-library/react": "^11.0.4",
+    "@types/loadable__component": "^5.13.2",
+    "monaco-editor-webpack-plugin": "1.9.1"
```

Comment:
`"monaco-editor-webpack-plugin": "1.9.1"` can be removed as it's not used anymore